fastchat/serve/inference.py

```diff
@@ -161,6 +162,12 @@ def generate_stream(
                                                       device=device))[0]
          start_ids = torch.as_tensor([[model.generation_config.decoder_start_token_id]],
                      dtype=torch.int64, device=device)
+    # Optionally defines logits processor
+    logits_processor = None
+    if "t5" in model.config._name_or_path:
```

Comment:
Instead of doing it specifically for T5, can we implement `RepetitionPenalty` as a general parameter similar to `temperature`?
We can move it to L142.
Then for T5, we can set the default value at these places similar to temperature
https://github.com/lm-sys/FastChat/blob/4f4637832981da418c064546a2ec3a12d6ab9399/fastchat/serve/gradio_web_server.py#L255

https://github.com/lm-sys/FastChat/blob/4f4637832981da418c064546a2ec3a12d6ab9399/fastchat/serve/inference.py#L322